### DISCUSSION TITLE
Codespace Creation

### DISCUSSION THREAD

**Discussion by bertoprez** [community/community#44711] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hi everyone,

I am trying to create a codespace but when I have to select one branch it keeps loading nothing, and I am not able to select absolutely nothing:

![image](https://user-images.githubusercontent.com/123019015/213279860-7e20aa8d-122f-4b6e-ada0-af6cb0a3763c.png)

Can you please tell me what's wrong?

Thanks in advance.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @bertoprez ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
